packages/core/components/Puck/components/Components/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { useComponentList } from "../../../../lib/use-component-list";
+import { useAppContext } from "../../context";
+import { ComponentList } from "../../../ComponentList";
+import { useMemo } from "react";
+
+export const Components = () => {
+  const { config, state, customUi } = useAppContext();
+
+  const componentList = useComponentList(config, state.ui);
+
+  const Wrapper = useMemo(() => customUi.componentList || "div", [customUi]);
```

Comment:
This component is called `<Puck.Components />`, but the `customUi` API refers to `componentList`, as do the other exported components `ComponentListDroppable` and `ComponentListDraggable`.

This might be fine, because the former is intended for composability and componentList is the internal name, but we may want to align somehow.